packages/testing-helpers/src/fixture-no-side-effect.js

```diff
@@ -5,6 +5,7 @@ import { isValidRenderArg } from './lib.js';
 /**
  * @typedef {object} FixtureOptions
  * @property {Element} [parentNode] optional parent node to render the fixture's template to
+ * @property {Object<string, typeof HTMLElement>} [scopedElements] optional scoped-elements definition map
```

Comment:
second type param will probably need to be a generic
something like 

```
@template {HTMLElement} T
@property {Object<string, T>}
```

But likely better expressed directly in TypeScript

Reply:
Honestly, I'm a noob regarding TypeScript 😅

First of all thanks for your comments @bennypowers 🤗

I'm not sure about that change because `typeof HTMLElement` refers to any class element that extends HTMLElement but using `@template {HTMLElement} T` I "think" that it refers to an instance of HTMLElement, not the class. Am I wrong?

Reply:
I created the types.d.ts file in `@open-wc/scoped-elements` to export the type
```typescript
export type ScopedElementMap = {
  [key: string]: typeof HTMLElement;
}
```